test/inductor/test_snode_runtime.py

```diff
@@ -2,21 +2,13 @@
 
 
 import torch
-from torch._dynamo.backends.registry import register_backend
 from torch._inductor import metrics
-from torch._inductor.compile_fx import compile_fx, count_bytes_inner
 from torch.testing._internal.common_utils import TestCase as TorchTestCase
 from torch.testing._internal.inductor_utils import HAS_CUDA
 
 aten = torch.ops.aten
 
 
-# TODO(xmfan): remove this backend once we call from prod i.e. @torch.compile codepath
```

Comment:
You lost this comment

Reply:
I didn't bring this comment because I removed the backend? I don't think it applies to the other definition tbh.

Reply:
its still under register_backend

Reply:
Yeah it was before too, but there isn't a comment to that on the original definition. It also doesn't entirely make sense, because if someone calls the torch.compile codepath in prod, this backend won't get registered anyway.